graphein/protein/tensor/io.py

```diff
@@ -254,6 +261,10 @@ def protein_to_pyg(
     )
     if store_het:
         out.hetatms = [het_coords]
+
+    if store_bfactor:
+        out.bfactor = torch.from_numpy(df["b_factor"].values)

```

Comment:
This will be of shape `num_atoms x` right? I expect this would break batching as all other tensors are of shape `n_res x X`

Reply:
Good catch, solved this via a group by now that averages b factors on a per residue basis, consistent with the plddt information in the predicted datasets.

Reply:
Ran more tests now and everything seems to work in a backward compatible way, lmk if there is anything else that needs to happen before merging